packages/psss/src/components/DateToolbar/DateToolbar.js

```diff
@@ -95,6 +96,7 @@ const MediumText = styled.span`
 `;
 
 export const DateToolbarComponent = ({ period, setPeriod }) => {
+  const isFetching = useIsFetching();
```

Comment:
If casting to boolean is required:

```suggestion
  const isFetching = !!useIsFetching();
```

and then you can use just `isFetching` in the code that follows

Reply:
Good idea. yes casting as a boolean is required. Just in case you are interested, the useIsFetching hook returns the number (hence the need to cast to a bool) of requests currently being fetched anywhere in the app which is great for global loading state.